waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -335,7 +359,9 @@ proc connectToNodes*(pm: PeerManager,
 
 # Ensures a healthy amount of connected relay peers
 proc relayConnectivityLoop*(pm: PeerManager) {.async.} =
-  while true:
+  pm.relayLoopUp = true
+  info "Starting relay connectivity loop"
+  while pm.relayLoopUp:
```

Comment:
...realising this may not be accessible from here. Perhaps then adding a `stop()` method and (private) `.started` `bool` to the PeerManager which gets formally stopped when we do `node.stop()`? I think these loops can at least be tied to the lifecycle of the peer manager and should not need to be tracked separately.

Reply:
added start() and stop()!